src/client/gui/gui_cmd.cpp

```diff
@@ -399,28 +399,25 @@ void cmd::GuiCmd::handle_petenv_instance(const google::protobuf::RepeatedPtrFiel
     // petenv doesn't exist yet
     if (petenv_instance == instances.cend())
     {
-        if (current_petenv_name != petenv_name)
-        {
-            petenv_start_action.setText("Start");
-            petenv_start_action.setEnabled(false);
-            petenv_shell_action.setEnabled(true);
-            petenv_stop_action.setEnabled(false);
-
-            current_petenv_name = petenv_name;
-        }
+        petenv_start_action.setText("Start");
+        petenv_start_action.setEnabled(false);
+        petenv_shell_action.setEnabled(false);
```

Comment:
i think this should remain `true`. so far, if there was no primary instance, clicking that button would launch a terminal and create the primary instance, and that's a nice one-click action that would put you into the shell of the primary instance